hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeManager.java

```diff
@@ -1281,6 +1282,57 @@ nodes with its data cleared (or user can just remove the StorageID
     }
   }
 
+  /**
+   * refresh the network topology of this cluster based on the mapping_topology.data file.
+   */
+  public void refreshTopology() throws IOException {
+    long start = System.currentTimeMillis();
```

Comment:
use Time.monotonicNow for duration calculations

Reply:
Sir, have fixed.